Trading/Mode/dca_trading_mode/dca_trading.py

```diff
@@ -191,10 +191,12 @@ async def create_new_orders(self, symbol, _, state, **kwargs):
                             (1 - multiplier) if side is trading_enums.TradeOrderSide.BUY else
                             (1 + multiplier)
                         )
-                        await self._create_entry_order(
+                        if not await self._create_entry_order(
                             secondary_order_type, secondary_quantity, secondary_target_price,
                             symbol_market, symbol, created_orders, price
-                        )
+                        ):
+                            # stop iterating if an order can't be created
```

Comment:
Why ? Should we stop without worrying about the reason why the order can't be created ?

Reply:
the reason is already logged from the `_create_entry_order` however i'll add a log to inform that other orders are also skipped. This info we don't log at the moment 👍

Reply:
Yes ok but I was asking if it's a good idea to stop iterating if without checking if it's due to an exchange error / timeout or a missing fund... The question is : should we stop without checking what's the error §

Reply:
At the moment if an error is raised, it will stop everything (it's not caught), this `break` statement only triggers when orders are either too small for exchange rules or if the max ratio is reached.
The logic behind it is that we don't try to create other orders if we can't create the 1st ones for reasons that will also prevent other ones.

Now we could add a try/except statement to "retry" failed orders on the exchange and/or continue creating other secondary entry orders. (this is not the current dca behavior)

Reply:
Ok I see. If this is the expected behavior we can let as it. I was wondering if we had taken into account the case when there is an issue with the exchange.